pontoon/messaging/templates/messaging/emails/notification_digest.html

```diff
@@ -0,0 +1,171 @@
+            background-color: #f6f6f6;
+        }
+
+        table.inner {
+            max-width: 600px;
+            padding: 70px 0;
+        }
+
+        table.inner a {
+            color: #F36;
+        }
+
+        .logo {
+            padding-bottom: 30px;
+        }
+
+        .title {
+            font-size: 36px;
+            margin: 20px 0;
+            padding-bottom: 10px;
+        }
+
+        .subtitle {
+            font-size: 15px;
+            margin: 10px 0 20px 0;
+            padding-bottom: 30px;
+        }
+
+        .footer {
+            font-size: 12px;
+            color: #888888;
+            margin-top: 20px;
+            padding-top: 10px;
+        }
+
+        /* Notifications */
+        .notifications {
+            background-color: #ffffff;
+            font-size: 15px;
+            padding: 0;
+            text-align: left;
+        }
+
+        .notifications ul {
+            list-style-type: none;
+            margin: 0;
+            padding: 0;
+        }
+
+        .notifications li {
+            margin: 0;
+        }
+
+        .notifications li.horizontal-separator {
+            border-top: 1px solid #f6f6f6;
+        }
+
+        .notifications li .item-content {
+            padding: 10px;
+        }
+
+        .notifications li .item-content .message {
+            color: #888888;
+            padding: 10px;
+        }
+
+        .notifications li .item-content .message p {
+            margin: 0;
+        }
+
+        .notifications li .item-content .description ul {
+            list-style: inside;
+            padding-top: 10px;
+        }
+
+        .notifications li .item-content .description ul li {
+            margin: 0;
+            padding: 2px 4px;
+        }
+
+        .notifications li .item-content .timeago {
+            color: #888888;
+            display: block;
+            font-size: 11px;
+            font-weight: normal;
+            margin-top: 8px;
+            text-align: right;
+            text-transform: uppercase;
+        }
+    </style>
+</head>
+
+<body>
+    <table class="outer">
+        <tr>
+            <td>
+                <table class="inner">
+                    <tr>
+                        <td class="logo">
+                            <a href="{{ full_url('pontoon.homepage') }}">
+                                <img src="{{ full_static('img/logo.png') }}" alt="Pontoon logo" width="52" height="64">
+                            </a>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="title">Hello.</td>
+                    </tr>
+                    <tr>
+                        <td class="subtitle">Here's a summary of the Pontoon notifications you've subscribed to.</td>
+                    </tr>
+                    <tr>
+                        <td class="notifications">
+                            {{ Notifications.list(notifications=notifications, no_title="No notifications.") }}
```

Comment:
If there are no notifications, shouldn't we skip sending the email?

Reply:
We do: 

https://github.com/mozilla/pontoon/pull/3456/files#diff-c2243faf007e8b605c8ddf8e45bd0e50196f1dea51fd740fe664b41c219fd4c9R52-R65

I'll remove the `no_title` parameter, as it is unused.